src/test/java/staffconnect/testutil/TypicalPersons.java

```diff
@@ -73,6 +66,20 @@ public class TypicalPersons {
             .withEmail("hans@example.com").withModule("CS2103T")
             .withFaculty("Computing").withVenue("chicago ave").build();
 
+    public static final Person KAFKA = new PersonBuilder().withName("Kafka Apache").withPhone("9452413")
+            .withEmail("apache@example.com").withModule("CS2102").withFaculty("Business")
+            .withVenue("pteruges avenue").withTags("classmate").withAvailabilities("FRIDAY").build();
+    public static final Person NATASHA = new PersonBuilder().withName("Natasha Harrower").withPhone("8019394")
+            .withEmail("harrower@example.com").withModule("CS2102")
+            .withFaculty("Computing").withVenue("underworld avenue")
+            .withTags("classmate").withAvailabilities("FRIDAY").build();
+    public static final Person LEONARDO = new PersonBuilder().withName("Leonardo DiCaprio").withPhone("88472234")
+            .withEmail("lcaprio@gmail.com").withModule("TS2237")
+            .withFaculty("Arts and Social Sciences").withVenue("LT13").build();
+    public static final Person MICHAEL = new PersonBuilder().withName("Michael Jackson").withPhone("92347123")
+            .withEmail("heehee@gmail.com").withModule("MUA1163")
+            .withFaculty("Music").withVenue("YSTCM-SR9").build();
```

Comment:
good move to add our manually added testing data here, forgot about CLARA though, next time can edit!

Reply:
Noted: #97